### ISSUE TITLE
Clarify the map view

### ISSUE THREAD

**Issue by joeroe** [xronos-ch/xronos.rails#80]
I was showing XRONOS to some colleagues today, and they had some useful feedback on the map view, seeing it for the first time:

* When you zoom in sufficiently, the green circles seem to "disappear", and it is not obvious that they have turned into red dots
* The black/red dots are hard to see against the base map
* In general, the OSM base map is quite "busy" and with no legend, it is confusing what all the symbols mean, and what is XRONOS vs. what is OSM.
* Labels in non-Latin languages are not particularly useful (obviously this is culturally-specific, but...)

I think switching to a more minimalist, subdued map theme could help a lot with this.

**Comment by joeroe** [comment]
In https://github.com/xronos-ch/xronos.rails/tree/map-style I've switched from using OSM's tileserver to a less busy base map via [leaflet-providers](https://github.com/leaflet-extras/leaflet-providers). I think this already really helps emphasise our data over the base map.

@MartinHinz suggested a satellite base map, so I've added that as an option too.

There are many, many options (see https://github.com/leaflet-extras/leaflet-providers#providers), I just picked a pair I liked, so I'm open to suggestions. Some require API keys but I don't think that should be a big deal, minus some possible billing issues (see xronos-ch/orga#101).
